packages/ia-components/sandbox/AnchorDetails.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import IAReactComponent from './IAReactComponent';
-const debug = require('debug')('dweb-archive:AnchorDetails');
+const debug = require('debug')('ia-components:AnchorDetails');
```

Comment:
can we use import statement?
how do we turn off when we use in prod?
maybe we need an entry in the README with a little bit of usage direction